integration-cli/docker_cli_events_test.go

```diff
@@ -66,7 +66,7 @@ func (s *DockerSuite) TestEventsContainerFailStartDie(c *check.C) {
 
 func (s *DockerSuite) TestEventsLimit(c *check.C) {
 	for i := 0; i < 30; i++ {
-		dockerCmd(c, "run", "busybox", "echo", strconv.Itoa(i))
+		go dockerCmd(c, "run", "--rm", "busybox", "true")
```

Comment:
Better synchronize everything with a channel to handle eventual error and I think another to handle docker events streaming checks